featuretools/tests/primitive_tests/test_agg_feats.py

```diff
@@ -200,6 +200,11 @@ def test_base_of_and_stack_on_heuristic(es, test_primitive):
     child.primitive.base_of = [test_primitive]
     assert check_stacking(test_primitive(), [child])
 
+    test_primitive.stack_on = []
+    child.primitive.base_of = []
+    child.primitive.base_of_exclude = [test_primitive]
+    assert not check_stacking(test_primitive(), [child])
```

Comment:
if `stack_on` and `base_of` are both set to `[]`. `check_stacking` will return False even if `test_primitive` is not in `base_of_exclude`

Reply:
Oh okay, interesting. I changed `stack_on` and `base_of` to None so they can return True. It kind of seems like whether `None` or `[]` they should return the same value.

Reply:
I think `None` means no whitelist while `[]` means an empty whitelist, so nothing allowed